app.yaml

```diff
@@ -0,0 +1,32 @@
+runtime: nodejs12
+
+handlers:
+  - url: /static/js/(.*)
+    static_files: build/static/js/\1
+    upload: build/static/js/(.*)
+    secure: always
+    redirect_http_response_code: 301
+  - url: /images/(.*\.(gif|png|jpg))$
+    static_files: public/images/\1
+    upload: public/images/.*\.(gif|png|jpg)$
+    secure: always
+    redirect_http_response_code: 301
+  - url: /(.*\.(json|ico|png|txt))$
+    static_files: build/\1
+    upload: build/.*\.(json|ico|png|txt)$
+    secure: always
+    redirect_http_response_code: 301
+  - url: /api(\/.*)*
+    script: auto
+    secure: always
+    redirect_http_response_code: 301
+  - url: /.*
+    static_files: build/index.html
+    upload: build/index.html
+    secure: always
+    redirect_http_response_code: 301
+
+# Expose this key is not a security problem since it is only accessible
```

Comment:
I implemented a strategy in which our secret keys are stored in Datastore, and injected to Node Runtime when we deploy to production.

This key is only the datastore ID for our secrets, so it is secure to expose it.